### PR TITLE
WEB-6348: Add iameffed.com flat files

### PR DESCRIPTION
**Pull request by thequicksilver**

### CODE REVIEW

File: dist/iameffed.com/global/css/ecards.css [hunk 1]
```diff
@@ -0,0 +1,339 @@
+@charset "UTF-8";
```
Comment (rediris) [inline dist/iameffed.com/global/css/ecards.css]: Beautify, please.

File: dist/iameffed.com/global/css/ecards_thankyou.css [hunk 2]
```diff
@@ -0,0 +1,336 @@
+@charset "UTF-8";
```
Comment (rediris) [inline dist/iameffed.com/global/css/ecards_thankyou.css]: Beautify, please.

File: dist/iameffed.com/global/js/ecards.js [hunk 3]
```diff
@@ -0,0 +1,329 @@
+// JavaScript Document
```
Comment (rediris) [inline dist/iameffed.com/global/js/ecards.js]: Beautify, please.

File: dist/iameffed.com/global/js/validation.js [hunk 4]
```diff
@@ -0,0 +1,506 @@
+function checkField(formName, fieldName, prettyText, required, requiredLength){
```
Comment (rediris) [inline dist/iameffed.com/global/js/validation.js]: Beautify, please.

File: dist/iameffed.com/home/thankyou.html [hunk 5]
```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```
Comment (rediris) [inline dist/iameffed.com/home/thankyou.html]: Beautify, please.

**Comment by rediris** [comment]
The `/home/index.html` mapping is a little weird. That's an older dotCMS thing, so we should just ditch the `/home/` bit.

**Comment by thequicksilver** [comment]
I mean, are we just backing these things up to archive them, or are we taking the time to clean up code in the process - in which case these should be much different tickets? The /home/ thing doesn't functionally impact anything in this case.

If we want to reorganize the files and code, I'm going to say make tickets specifically for that work separately, as it's not related to the backing up process.

**Comment by rediris** [comment]
@thequicksilver the index.html file in `/home/` is the site's index page as mapped by dotCMS. If we don't deal with it now, the site won't function as expected, which is part of the expected outcome of this ticket.

**Pull request merged**